MiniShopping/MiniShopping/Controllers/ProductsViewController.m

```diff
@@ -0,0 +1,86 @@
+#import "ProductsViewController.h"
+#import "ProductsView.h"
+#import "ProductCell.h"
+#import "Product.h"
+#import "FetchProducts.h"
+#import "NSMutableArray+Sorting.h"
+
+
+@interface ProductsViewController () <UICollectionViewDelegate, UICollectionViewDataSource, FetchProductsDelegate, ProductViewDelegate> {
+    NSMutableArray<Product *> *products;
+    ProductsView *productsView;
+    FetchProducts *fetchProducts;
+}
+
+@end
+
+@implementation ProductsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.title = @"Mini Shopping";
+    self.view.backgroundColor = UIColor.systemBackgroundColor;
+    [self setupModel];
+    [self setupView];
+}
+
+- (void)setupModel {
+    products = [[NSMutableArray alloc] init];
+    fetchProducts = [[FetchProducts alloc] init];
+    [fetchProducts fetchData];
+    fetchProducts.delegate = self;
+}
+
+- (void)setupView {
+    productsView = [[ProductsView alloc] initWithFrame:CGRectZero];
+    productsView.delegate = self;
+    [productsView.productsGridView registerClass:[ProductCell class] forCellWithReuseIdentifier:@"productsGridViewIdentifier"];
+    productsView.productsGridView.translatesAutoresizingMaskIntoConstraints = NO;
+    productsView.productsGridView.delegate = self;
+    productsView.productsGridView.dataSource = self;
+    [self.view addSubview:productsView];
+    
+    productsView.translatesAutoresizingMaskIntoConstraints = NO;
+    [productsView.topAnchor constraintEqualToAnchor:self.view.safeAreaLayoutGuide.topAnchor].active = YES;
+    [productsView.leadingAnchor constraintEqualToAnchor:self.view.leadingAnchor].active = YES;
+    [productsView.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor].active = YES;
+    [productsView.bottomAnchor constraintEqualToAnchor:self.view.safeAreaLayoutGuide.bottomAnchor].active = YES;
+}
+
+- (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView {
+    return 1;
+}
+
+- (NSInteger)collectionView:(nonnull UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
+    return products.count;
+}
+
+- (nonnull __kindof UICollectionViewCell *)collectionView:(nonnull UICollectionView *)collectionView cellForItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
```

Comment:
Move the method parameters into new line and align them properly. This will make to code more readable.

The code/statement line should not longer than 100 characters, if needed move parameters into new line.